src/main/scala/org/deepdive/ddlog/DeepDiveLogParser.scala

```diff
@@ -224,9 +240,7 @@ class DeepDiveLogParser extends JavaTokenParsers {
     | bcond
     )
   def bcond : Parser[Cond] =
-    ( expr ~ compareOperator ~ expr ^^ { case (lhs ~ op ~ rhs) =>
-        ComparisonCond(lhs, op, rhs)
-      }
+    ( cexpr ^^ ExprCond
```

Comment:
`cexpr` can be a single `lexpr`, but some expressions are not conditions. For example, we will have a problem compiling `R(x) :- S(x), 1.`.